### PR TITLE
[4.0] Header improvements

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

This PR merges the to header rows into 1, therefore more giving the user more space for main content. It also deals with the header issue mentioned in https://github.com/joomla/joomla-cms/issues/15392

@ciar4n @dgt41 @armenos

untitled-1

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on edcb2ef3de00129bf75646df06822a25e50aeff1<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15404">issues.joomla.org/tracker/joomla-cms/15404</a>.</sub>

**Review by ciar4n — APPROVED** [review]

**Comment by wilsonge** [comment]
Conflicts please :)

**Comment by C-Lodder** [comment]
@wilsonge  - done

File: administrator/templates/atum/index.php [hunk 1]
```diff
@@ -92,22 +92,32 @@
...
-						<a id="menu-collapse" class="menu-toggle" href="#">
-							<span class="fa fa-bars fa-fw">
-								<span class="sr-only"><?php echo JText::_('TPL_ATUM_CONTROL_PANEL_MENU'); ?></span>
-							</span>
-						</a>
+				<div class="d-flex justify-content-between align-items-center">
+					<div class="d-flex justify-content-start align-items-center">
+						<div class="menu-collapse hidden-lg-up">
+							<a id="menu-collapse" class="menu-toggle" href="#">
+								<span class="fa fa-bars fa-fw">
```
Comment (brianteeman) [inline administrator/templates/atum/index.php]: shouldnt this have aria-hidden=true
?

Comment (C-Lodder) [inline administrator/templates/atum/index.php]: I don't know....should it?

Comment (brianteeman) [inline administrator/templates/atum/index.php]: my belief is that it should be to make sure that the screenreader doesnt try to announce the font value and instead just reads the text you have set for the screenreader

Comment (wilsonge) [inline administrator/templates/atum/index.php]: According to their website yes http://fontawesome.io/accessibility/

Comment (C-Lodder) [inline administrator/templates/atum/index.php]: done

**Comment by rjcf18** [comment]
I have tested this item :white_check_mark: successfully on e9c09645f5daf3b67e6faee98995043cc09ab9ed<br /><br />Seems to be working great:

Desktop:
deepinscreenshot20170420130054

Tablet/mobile:
deepinscreenshot20170420130143
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15404">issues.joomla.org/tracker/joomla-cms/15404</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on e9c09645f5daf3b67e6faee98995043cc09ab9ed<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15404">issues.joomla.org/tracker/joomla-cms/15404</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Pull request merged**